### PR TITLE
Show account public keys in more places

### PR DESCRIPTION
**Pull request by buoyad**
Creates a new common component `SmallAccountID` that initially displays `XXXXXXXX...XXXXXXXX` and expands and becomes selectable on click. 
smallaccountid
This adds `SmallAccountID` to the wallet header and sender/recipient in transaction details. Also adds the accountID in wallet settings in a `CopyText`. r? @keybase/react-hackers

### CODE REVIEW

**Comment by buoyad** [comment]
@keybase/react-hackers bump

**Comment by chrisnojima** [comment]
i guess my reply didn't go through. i believe the UX is that its supposed to expand on hover, not click
cc: @cecileboucheron

**Comment by cecileboucheron** [comment]
This could work too I think. Especially because it'll be that exact behavior on mobile. 
@malgorithms @adamjspooner Thoughts?

**Comment by buoyad** [comment]
@malgorithms and I talked a bit about this last week - a pro of this approach is we don't have to handle the case where the user click-and-drags outside the hover area and causes the ID to collapse

**Comment by malgorithms** [comment]
yeah, I prefer click, rather than hover

File: shared/wallets/wallet/header/container.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ const mapStateToProps = (state: TypedState) => {
   return {
     accountID: selectedAccount.accountID,
     isDefaultWallet: selectedAccount.isDefault,
-    keybaseUser: state.config.username,
+    keybaseUser: state.config.username || '',
```
Comment (chrisnojima) [inline shared/wallets/wallet/header/container.js]: username is actually always a string now, if its empty its ''

**Pull request merged**